scripts/release/tests/test-release.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+pip3 install -r scripts/release/requirements.txt
+pip3 list
```

Comment:
I just added this to try to see what was going on on the build agent - sure enough it says pytest is installed!